src/group/pages/EditPlaceTypes.spec.js

```diff
@@ -0,0 +1,89 @@
+import '@testing-library/jest-dom'
+import userEvent from '@testing-library/user-event'
+import { render } from '@testing-library/vue'
+import { flushPromises } from '@vue/test-utils'
+import { Dialog } from 'quasar'
+
+import { resetServices } from '@/utils/datastore/helpers'
+
+import { withDefaults } from '>/helpers'
+import {
+  useMockBackend,
+  createUser,
+  createGroup,
+  createPlaceType,
+  loginAs,
+} from '>/mockBackend'
+import { addUserToGroup } from '>/mockBackend/groups'
+import '>/routerMocks'
+
+import EditPlaceTypes from './EditPlaceTypes'
+
+Dialog.create = jest.fn(() => {
+  return { onOk (fn) { fn('') } }
+})
+
+describe('EditPlaceTypes', () => {
+  useMockBackend()
+  let group
+
+  beforeEach(() => {
+    jest.resetModules()
+    resetServices()
+  })
+
+  beforeEach(() => {
+    const user = createUser()
+    group = createGroup()
+    addUserToGroup(user, group)
+    user.currentGroup = group.id
+    loginAs(user)
+  })
+
+  it('renders place types edit form', async () => {
+    const { findByText } = render(EditPlaceTypes, withDefaults())
+    await flushPromises()
+
+    await findByText('Name')
+  })
+
+  it('adds new place type', async () => {
+    const { click, type } = userEvent.setup()
+
+    const { queryByRole, findByTitle, findByRole, findByText } = render(EditPlaceTypes, withDefaults())
+    await flushPromises()
+
+    await click(await findByTitle('Add new place type'))
+    const textbox = await findByRole('combobox', { name: 'Name' })
+    // somehow we need to type the name before we can choose it
+    // seems like a problem between jsdom/testing-library and Quasar
```

Comment:
My guess is that because the select is closed at first, you need to click the `findByRole('combobox', { name: 'Name' })` to open it... then can click the Distribution Place within it?

Reply:
the comment is also copy-pasta and likely wrong. in this case we really need to type the name as it's not a predefined option.